src/Notepads/Strings/de-DE/Settings.resw

```diff
@@ -186,11 +186,11 @@
     <comment>PersonalizationPage AccentColorSettings UseWindowsAccentColorToggleSwitch On display text.</comment>
   </data>
   <data name="PersonalizationPage_BackgroundTintOpacitySettings_Description.Text" xml:space="preserve">
-    <value>Acryl Hintergrundfarbton Transparenz. Please note that Acrylic effect will be disabled when battery saver mode is on or if you turn off transparency effects in Windows Settings.</value>
+    <value>Opazität der Acryl-Hintergrundfarbe. Der Acryleffekt wird deaktiviert wird, wenn der Batteriesparmodus eingeschaltet ist oder wenn die Transparenzeffekte in den Windows-Einstellungen ausschaltet sind.</value>
```

Comment:
> Der Acryleffekt wird deaktiviert wird,

ein "wird" zu viel.
Besser:
Der Acryleffekt wird deaktiviert,

Reply:
I will fix it.

Reply:
https://github.com/JasonStein/Notepads/commit/f886a1cfdc618dcc51f03cf4350e680aefc346ba

Reply:
> [f886a1c](https://github.com/JasonStein/Notepads/commit/f886a1cfdc618dcc51f03cf4350e680aefc346ba)

Fix looks good 👍

Reply:
sorry, not quite ... at the end of the string I also oversaw "ausschaltet" (that's wrong).
This is how the string should look like:

Opazität der Acryl-Hintergrundfarbe. Der Acryleffekt wird deaktiviert, wenn der Batteriesparmodus eingeschaltet ist oder wenn die Transparenzeffekte in den Windows-Einstellungen ausgeschaltet sind.

Sorry again ...

Reply:
> > [f886a1c](https://github.com/JasonStein/Notepads/commit/f886a1cfdc618dcc51f03cf4350e680aefc346ba)
> 
> Fix looks good 👍

Any comments on @milotype's suggestion?

Reply:
I oversaw that too but the new fix solve that

Reply:
So what should I use here?

Reply:
Opazität der Acryl-Hintergrundfarbe. Der Acryleffekt wird deaktiviert, wenn der Batteriesparmodus eingeschaltet ist oder wenn die Transparenzeffekte in den Windows-Einstellungen ausgeschaltet sind.